ceph.yaml

```diff
@@ -0,0 +1,342 @@
+      - libxml2-dev
+      - lttng-ust-tools
+      - lua5.4-dev
+      - lz4-dev
+      - make
+      - nasm
+      - ncurses-dev
+      - ninja
+      - nodejs
+      - npm
+      - oath-toolkit-dev
+      - openldap-dev
+      - openssl-dev
+      - pkgconf
+      - posix-libc-utils
+      - py${{vars.py-version}}-cython
+      - py${{vars.py-version}}-prettytable
+      - py${{vars.py-version}}-pyyaml
+      - py${{vars.py-version}}-setuptools
+      - py${{vars.py-version}}-sphinx-bin
+      - python-${{vars.py-version}}-dev
+      - re2-dev
+      - snappy-dev
+      - swagger
+      - systemd-dev
+      - thrift-dev
+      - util-linux-dev
+      - wolfi-base
+      - xfs-scrub
+      - xfsprogs
+      - yaml-cpp-dev
+      - zlib-dev
+      - zstd-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/ceph/ceph.git
+      tag: v${{package.version}}
+      expected-commit: 58a7fab8be0a062d730ad7da874972fd3fba59fb
+      recurse-submodules: true
+
+  - uses: patch
+    with:
+      patches: fix-submodule.patch fix-osd-class-dir.patch
+
+  - uses: cmake/configure
+    with:
+      opts: |
+        -Wno-dev \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_INSTALL_PREFIX=/usr \
+        -DCMAKE_CXX_STANDARD=17 \
+        -DCMAKE_INSTALL_LOCALSTATEDIR=/var \
+        -DPython3_LIBRARIES=/usr/lib/python3.11 \
+        -DCMAKE_INSTALL_SYSCONFDIR=/etc \
+        -DNINJA_MAX_COMPILE_JOBS=$(nproc) \
+        -DWITH_GTEST_PARALLEL=ON \
+        -DWITH_PYTHON3=${{vars.py-version}} \
+        -DENABLE_GIT_VERSION=ON \
+        -DLUA_LIBRARIES=/usr/lib/lua5.4/liblua.so \
+        -DWITH_BABELTRACE=OFF \
+        -DWITH_JAEGER=OFF \
+        -DWITH_LTTNG=OFF \
+        -DWITH_MANPAGE=ON \
+        -DWITH_RADOSGW_AMQP_ENDPOINT=OFF \
+        -DWITH_RADOSGW_SELECT_PARQUET=ON \
+        -DWITH_RADOSGW_POSIX=OFF \
+        -DWITH_RDMA=OFF \
+        -DWITH_REENTRANT_STRSIGNAL=ON \
+        -DWITH_SEASTAR=OFF \
+        -DWITH_SPDK=OFF \
+        -DWITH_SYSTEMD=ON \
+        -DWITH_CEPHFS_SHELL=ON \
+        -DWITH_SYSTEM_ARROW=ON \
+        -DWITH_SYSTEM_BOOST=OFF \
+        -DWITH_SYSTEM_FMT=OFF \
+        -DWITH_SYSTEM_LIBURING=ON \
+        -DWITH_SYSTEM_NPM=ON \
+        -DWITH_MGR_DASHBOARD_FRONTEND=ON \
+        -DWITH_SYSTEM_ROCKSDB=OFF \
+        -DWITH_SYSTEM_ZSTD=ON \
+        -DWITH_QATLIB=OFF \
+        -DWITH_QATZIP=OFF \
+        -DWITH_TESTS=OFF \
+        -DWITH_THREAD_SAFE_RES_QUERY=ON
+
+  - runs: cmake --build output --target common-options-objs
+
+  - uses: cmake/build
+
+  - uses: cmake/install
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-dev
+    dependencies:
+      runtime:
+        - ${{package.name}}
+    pipeline:
+      - uses: split/dev
+    test:
+      pipeline:
+        - uses: test/tw/ldd-check
+
+  - name: ${{package.name}}-libs
+    description: "libraries for ${{package.name}}"
+    dependencies:
+      provides:
+        - so:libceph-common.so.2=2
```

Comment:
This should be generated by melange

Reply:
@EyeCantCU @javacruft This lib is found but is not packaged by melange as a runtime dep.

Reply:
Removed dependency and moved library module to `/usr/lib`